string objectName
uint32 x
uint32 y
uint32 z
uint32 oriented_x
uint32 oriented_y
uint32 oriented_z


